include/deal.II/base/parameter_handler.h

```diff
@@ -1127,6 +1127,25 @@ class ParameterHandler : public Subscriptor
                 const std::string           &documentation = "",
                 const bool                   has_to_be_set = false);
 
+  /**
+   * Mark a previously declared parameter as deprecated. This will cause an
+   * exception of type ExcEncounteredDeprecatedEntries to be thrown if
+   * the parameter is used in the input file.
+   *
+   * The exception message will list the name of the parameter.
```

Comment:
To make clear that there may be multiple such parameters:
```suggestion
   * Mark a previously declared parameter as deprecated. This will cause an
   * exception of type ExcEncounteredDeprecatedEntries to be thrown if
   * the parameter is used in the input file.
   *
   * The exception message will list the name of the parameter(s) that were encountered
   * in the input file.
```